app/controllers/hosts_controller.rb

```diff
@@ -20,11 +20,17 @@ class HostsController < ApplicationController
                         select_multiple_puppet_proxy update_multiple_puppet_proxy
                         select_multiple_puppet_ca_proxy update_multiple_puppet_ca_proxy)
 
+  HOST_POWER = {
+    :on => "on",
+    :off => "off",
+    :na => "na"
+  }.freeze
```

Comment:
i was intrigued by this comment, because it is a linter-suggestion, but I never questioned it. The strings will not be frozen, you [can freeze them though of course](https://gist.github.com/d33c19c5e6eb9906c44d6d8c1df9616e), and in this case I would argue in favour of freezing, since it is assigning to a constant and constants should never change. [This](http://flats.github.io/blog/2016/01/03/frozen-strings/) article also describes freezing as the "low-hanging fruit of Ruby optimization"